src/engraving/rw/write/writer.h

```diff
@@ -27,8 +27,8 @@
 #include "writecontext.h"
 #include "../compat/writescorehook.h"
 
-namespace mu::engraving::rw400 {
-class Write400 : public rw::IWriter
+namespace mu::engraving::write {
+class Writer : public rw::IWriter
```

Comment:
You forgot to rename this define MU_ENGRAVING_WRITE400_H

Reply:
Thanks, I will fix it in next PR